Add documentation for SessionNotCreatedException in troubleshooting errors

yvsvarma opened the pull request:
### **User description**
**Description**
This PR adds documentation for the `SessionNotCreatedException` under the "Troubleshooting Errors" section of the Selenium site. It includes a concise explanation of the exception, its likely causes, and actionable solutions.

**Motivation and Context**
The `SessionNotCreatedException` is a common error encountered by users, often caused by browser-WebDriver version mismatches, macOS privacy settings, or permission issues on the WebDriver binary. Documenting this will help users quickly understand and resolve the issue, improving their experience with Selenium.

**Types of Changes**
 Documentation improvement (added a new section under "Troubleshooting Errors").

**Checklist**
 I have read the contributing guidelines.
 I have tested the changes locally using [Hugo](https://gohugo.io/).
 I have ensured the documentation renders as expected and the format is consistent with other entries.


___

### **PR Type**
Documentation


___

### **Description**
- Added documentation for `SessionNotCreatedException` under "Troubleshooting Errors".

- Explained common causes of the exception, including version mismatches and system restrictions.

- Provided actionable solutions to resolve the exception, such as matching WebDriver and browser versions.

- Included macOS-specific guidance and Linux/macOS permission fixes.


___



### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>_index.en.md</strong><dd><code>Documented `SessionNotCreatedException` causes and solutions</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/troubleshooting/errors/_index.en.md

<li>Added a new section for <code>SessionNotCreatedException</code>.<br> <li> Described likely causes of the exception.<br> <li> Provided detailed solutions for resolving the issue.<br> <li> Included platform-specific troubleshooting steps.


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2119/files#diff-d8713bd094ba75ca3ce0659e13a3158956619dcbb35d5ede1743e90dbee52f9a">+18/-0</a>&nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

yvsvarma committed:
Add documentation for SessionNotCreatedException in troubleshooting errors

yvsvarma committed:
Merge branch 'trunk' into docs/session-not-created-exception

harsha509 reviewed (changes requested):
HI @yvsvarma ,

Thank you for the PR. Could you please add similar changes in other translated pages too. (zh-cn, pt-br, ja)

Thanks,
Sri

yvsvarma commented:
Sure @harsha509 , I will update the same in other languages too and will update! Thank you very much for reviewing this 😊

yvsvarma committed:
Add SessionNotCreatedException to Chinese translation (_index.zh-cn.md)

yvsvarma committed:
Add SessionNotCreatedException to Portuguese translation (_index.pt-br.md)

yvsvarma commented:
> HI @yvsvarma ,
> 
> Thank you for the PR. Could you please add similar changes in other translated pages too. (zh-cn, pt-br, ja)
> 
> Thanks, Sri

Hi Sri,
I updated the content in other language files too. 
Chinese - I added in Chinese language only.
For Ja & pt-br - The existing files are in English only.  Hence added english version in those files.

Please review.

Thanks,
Vinay

harsha509 reviewed (approved):
Thank you @yvsvarma !

harsha509 committed:
Merge branch 'trunk' into docs/session-not-created-exception

yvsvarma commented:
> Thank you @yvsvarma !

Thanks again for your time in reviewing this Sri!
